packages/use-dataloader/src/dataloader.ts

```diff
@@ -52,6 +58,17 @@ class DataLoader<T = unknown> {
     this.keepPreviousData = args?.keepPreviousData
     this.notify = args.notify
     this.maxDataLifetime = args.maxDataLifetime
+    if (args.enabled) {
+      const tryLaunch = () => {
+        if (DataLoader.started < DataLoader.maxConcurrent) {
+          // eslint-disable-next-line no-void
+          void this.load()
```

Comment:
I don't get why `void` is for here?

Reply:
load is async and promise shouldn't be launched without then/await according to our rules so by putting void it says that this is wanted

Reply:
not obvious, I'd prefer an explicit comment.